python/ray/tests/BUILD

```diff
@@ -45,7 +45,7 @@ py_test_module_list(
     "test_memstat.py",
     "test_mldataset.py",
   ],
-  size = "medium",
+  size = "large",
   extra_srcs = SRCS,
   tags = ["exclusive", "medium_size_python_tests_a_to_j", "team:core"],
```

Comment:
strictly speaking, should this now become `large_size_python_tests_shard_N`, where N is one of 0, 1, 2?